### PR TITLE
Update NSS 3.93

### PR DESCRIPTION
**Pull request by skhamis**
### Pull Request checklist ###

- **Breaking changes**:  This PR follows our [breaking change policy](https://github.com/mozilla/application-services/blob/main/docs/howtos/breaking-changes.md)
  - [ ] This PR follows the breaking change policy:
     - This PR has no breaking API changes, or
     - There are corresponding PRs for our consumer applications that resolve the breaking changes and have been approved
- [ ] **Quality**: This PR builds and tests run cleanly
  - Note:
    - For changes that need extra cross-platform testing, consider adding `[ci full]` to the PR title.
    - If this pull request includes a breaking change, consider [cutting a new release](https://github.com/mozilla/application-services/blob/main/docs/howtos/cut-a-new-release.md) after merging.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry in [CHANGELOG.md](../CHANGELOG.md) or an explanation of why it does not need one
  - Any breaking changes to Swift or Kotlin binding APIs are noted explicitly
- [ ] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/main/docs/dependency-management.md)
  - Any new dependencies are accompanied by a summary of the due dilligence applied in selecting them.

[Branch builds](https://github.com/mozilla/application-services/blob/main/docs/howtos/branch-builds.md): add `[firefox-android: branch-name]` to the PR title.

### CODE REVIEW

**Review by moztcampbell — APPROVED** [review]

File: libs/build-nss-desktop.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ fi
 if [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
   #From https://firefox-ci-tc.services.mozilla.com/tasks/index/app-services.cache.level-3.content.v1.nss-artifact/latest
   curl -sfSL --retry 5 --retry-delay 10 -O "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/app-services.cache.level-3.content.v1.nss-artifact.latest/artifacts/public/dist.tar.bz2"
-  SHA256="9573e640797200891dadc6401cdd9cb06e3fd48518a146cf2f578e83127cb7f2"
+  SHA256="15e8cab63a4e0484564231a24d23d8be349217ba6add53b56609c339bb306829"
   echo "${SHA256}  dist.tar.bz2" | shasum -a 256 -c - || exit 2
```
Comment (bendk) [inline libs/build-nss-desktop.sh:63]: I guess the task is failing because the index hasn't been updated yet?

**Review by bendk — APPROVED** [review]
I see a failing task, but maybe that's okay?  I can't wait to have a better process for this.

File: libs/build-nss-desktop.sh (hunk 1 above)
Comment (bendk) [reply libs/build-nss-desktop.sh:63]: I think the long-term solution to this is get in the monorepo.  In the meantime, what if we added support for overriding the URL via an ENV variable?  Then when we're running level 1 builds, we use the level 1 index.

**Review by moztcampbell — CHANGES_REQUESTED** [review]
build-all.sh needs updates or linux will keep using the existing NSS version

**Comment by skhamis** [comment]
https://github.com/mozilla/application-services/pull/5798 supersedes this

**Pull request closed without merging**